packages/create-sitecore-jss/src/initializers/angular/prompts.ts

```diff
@@ -31,5 +31,6 @@ export const prompts = [
       return true;
     },
   },
+  ...clientAppPrompts,
   ...styleguidePrompts,
```

Comment:
It seems like that question related to "language" is still present, while it's not expected to be present in "angular" and "angular-xmcloud" templates